lib/puppet/provider/zone/solaris.rb

```diff
@@ -234,10 +234,10 @@ def getconfig
           end
           current[$1.intern] = $2
         else
-          err "Ignoring '#{line}'"
+          Puppet.err "Ignoring '#{line}'"
```

Comment:
Well, we can't even be sure that the path/tags/file/line methods for providers are the RIGHT methods to call. A developer can make a custom provider with any one of those methods and end up getting unexpected calls, right? It seems like some sort of manifest source callback would be needed that can be referenced instead of the provider methods directly.